core/src/test/java/org/apache/sedona/core/spatialOperator/JoinQueryDeduplicationTest.java

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
Could you please add another test in Sedona Spark SQL to verify that this bug is eliminated?

Reply:
Sure! Hopefully I have time to add it tomorrow